src/main/java/com/stripe/model/AccountCreateParams.java

```diff
@@ -0,0 +1,182 @@
+package com.stripe.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class AccountCreateParams {
+
+  private String type;
```

Comment:
I think typically with a builder you want to make the class being built immutable - I would make these `final` and drop the `@Setter` above